pubspec.yaml

```diff
@@ -15,7 +15,7 @@ homepage: https://github.com/Workiva/w_flux
 dependencies:
   meta: ^1.0.4
   react: '>=3.0.0 <5.0.0'
-  w_common: ^1.7.0
+  w_common: ^1.9.0
```

Comment:
updated minimum w_common to 1.9.0 the first version that has `isOrWillBeDisposed`